megamek/src/megamek/common/MechSummary.java

```diff
@@ -307,6 +316,22 @@ public boolean getMilitary() {
         return military;
     }
 
+    public boolean getIndustrial() {
```

Comment:
MechSummary already has isIndustrialMek(), isSupportVehicle() and isAerospaceFighter(). 
isNaval() can be added to BTObject and implemented in ASCardDisplayable as movement code = "n" and in Tank there is a similar method that can be forwarded to.

Reply:
MechSummary, AlphaStrikeElement and Entity all implement BTObject. I prefer isX() methods over ETYPE checks as methods have a comment that tells me what they return while with the ETYPE checks I never know for which units it'll come out as true without looking exactly where they are used.

Reply:
i did it as an etype so that, i could get it to OR with the other ones correctly.

Reply:
I don't understand. I see that you are using 
entityTypes = entityTypes | Entity.ETYPE_AEROSPACEFIGHTER
etc. and that's ok. 
But you want to add
public boolean getSupportVehicle() {
        return supportVehicle;
    }
to MechSummary when it already inherits the method isSupportVehicle which does exactly the same and ditto for AerospaceFighter and IndustrialMek. I fail to see the gain or where the existing methods couldn't be used in the place of the new ones.

Reply:
wow this throws me, I didn't want to add a second review...

Reply:
thanks, this make it more clear what you asking.